packages/jest-reporters/src/GitHubActionsReporter.ts

```diff
@@ -5,55 +5,132 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {
+  summary as actionsSummary,
+  error as errorAnnotation,
+} from '@actions/core';
 import stripAnsi = require('strip-ansi');
 import type {
   AggregatedResult,
+  Test,
+  TestCaseResult,
   TestContext,
-  TestResult,
 } from '@jest/test-result';
+import {
+  getStackTraceLines,
+  getTopFrame,
+  separateMessageFromStack,
+} from 'jest-message-util';
+import {pluralize} from 'jest-util';
 import BaseReporter from './BaseReporter';
 
-const lineAndColumnInStackTrace = /^.*?:([0-9]+):([0-9]+).*$/;
-
-function replaceEntities(s: string): string {
-  // https://github.com/actions/toolkit/blob/b4639928698a6bfe1c4bdae4b2bfdad1cb75016d/packages/core/src/command.ts#L80-L85
-  const substitutions: Array<[RegExp, string]> = [
-    [/%/g, '%25'],
-    [/\r/g, '%0D'],
-    [/\n/g, '%0A'],
-  ];
-  return substitutions.reduce((acc, sub) => acc.replace(...sub), s);
-}
+const ancestrySeparator = ' \u203A ';
```

Comment:
do we need this when we depend on`@actions/core`? I'd have thought it contained all the escape sequences needed

Reply:
Right, `@actions/core` does all escapes. This is `>` for full test title: `describe > some test`. Const name could be better